backend/src/data/repositories/user-details/user-details.repository.ts

```diff
@@ -61,6 +62,25 @@ class UserDetails {
       .execute();
   }
 
+  public async updateMoneyBalance(
+    userId: number,
+    newMoneyBalance: number,
+  ): Promise<UserDetailsWithMoneyBalanceDto> {
+    const details = await this.#UserDetailsModel
+      .query()
+      .select()
+      .where('userId', userId)
```

Comment:
Can't you just?
```suggestion
      .where('userId', userId)
      .patchAndFetch({
        moneyBalance: newMoneyBalance,
      })
```

Reply:
Nope, In that case the exception will be thrown, because for patchAndFetch required entity (and here entity is represented as details)

Reply:
![image](https://user-images.githubusercontent.com/88377450/190220903-3ca3fffd-3da0-4940-abf4-3ea1d7d9b960.png)

Reply:
I have changed the approach and avoided 2 requests 👍